event_sourcery_sqlalchemy/sqlalchemy_event_store.py

```diff
@@ -118,13 +145,18 @@ def ensure_stream(self, stream_id: StreamId, expected_version: int) -> None:
                     StreamModel.uuid == stream_id,
                     StreamModel.version == expected_version,
                 )
-                .values(version=StreamModel.version + 1)
+                .values(
+                    version=StreamModel.version + 1
+                )  # TODO: may not be true! for inserting multiple events at once
             )
             result = self._session.execute(stmt)
 
             if result.rowcount != 1:  # optimistic lock failed
                 raise ConcurrentStreamWriteError
 
+        assert stream_id is not None
```

Comment:
Mypy couldn't know if `stream_id` can be null by now
(yeah, I know - in Rust it wouldn't compile :D )